requirements.test.txt

```diff
@@ -0,0 +1,4 @@
+ck==19.3b0 ; python_version > '3.5'
```

Comment:
What uses this flie?

Reply:
@gundalow , We added this file to list the packages to be pre-installed to run the sanity test for this collection. This should help developers run sanity test before submitting any change. Do you think it is not necessary ?

Reply:
If you run `ansible-test sanity --venv` or
`ansible-test sanity --docker` the requirements file under test/sanity will be automatically installed. 

See community.general for an example https://github.com/ansible-collections/community.general/blob/master/tests/unit/requirements.txt

Reply:
So when you add `5` from https://github.com/ansible-collections/dellemc.os10/issues/20 you will have CI triggered on every PR to this repo.

Reply:
@gundalow ,thanks for the clarification.I have removed the requirements.test.txt now and have added the requiremtns.txt under tests/sanity directory. Now, I get error no module named "ansible.netcommon". How do we have ansible.netcommon installed,when running automated sanity tests for pull-requests ?

Reply:
`ansible-galaxy collection install ansible.netcommon -p ../../`

Reply:
Added "ansible-galaxy collection install ansible.netcommon -p ../../" to the tests/sanity/requirements.txt, but it did not help, still getting the same error

Reply:
Apologies, this should be a new step in the workflow after ansible has been installed.

Reply:
@gundalow , to clarify, when I install netcommon and run "ansible-test sanity --docker" locally, there are no issues.
The error happens only for the sanity test triggered automatically on the Pull-requests in github. 
Is there anyway , I can force installation of netcommon, before the automatic test run ?

Reply:
After adding relevant details to the ansible-test.yml, this works fine.Thanks.